dome9/common/providerconst/const.go

```diff
@@ -115,6 +115,7 @@ const (
 	AP_SOUTHEAST_4	    = "32"
 	EU_CENTRAL_2	    = "33"
 	EU_SOUTH_2	    = "34"
+	IL_CENTRAL_1 =  "35"
```

Comment:
please add spaces to align with other region's

Reply:
fixed